pkgs/by-name/pa/packetry/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  libusb1,
+  glib,
+  gdk-pixbuf,
+  cairo,
+  pango,
+  gtk4,
+  graphene,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "packetry";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "greatscottgadgets";
+    repo = pname;
```

Please replace uses of `pname` with the actual package name, for further details please see #277994.
```suggestion
    repo = "packetry";
```